vim-patch:partial 53f7fccc9413 (#15631)
neovim/neovim: runtime/doc/autocmd.txt

				mode or to another window.  Also when the text
				of the cursor line has been changed, e.g. with
				"x", "rx" or "p".
				Not triggered when there is typeahead, while
				executing commands in a script file, when

> 				another window while remaining at the same

Reviewer: wording looks off, should be "when moving to..."

> 				Not triggered when there is typeahead, while
> 				executing commands in a script file, when
> 				another window while remaining at the same

Reviewer: - wording looks off, should be "when moving to..."
- "executing commands in a script file" is pedantic, can be condensed to "executing a script"

```suggestion
				Not triggered when there is typeahead, while
				executing a script, or when moving to
				another window while remaining at the same
```
